x-pack/plugins/apm/common/anomaly_detection.ts

```diff
@@ -40,6 +40,20 @@ export function getSeverity(score?: number) {
 }
 
 export function getSeverityColor(theme: EuiTheme, severity?: Severity) {
+  switch (severity) {
+    case Severity.warning:
+      return theme.eui.euiColorVis0;
+    case Severity.minor:
+    case Severity.major:
+      return theme.eui.euiColorVis5;
+    case Severity.critical:
+      return theme.eui.euiColorVis9;
+    default:
+      return;
+  }
+}
```

Comment:
You can also use helper function from ml plugin to get severity color, this will make sure consistency between plugins.

`import { getSeverityColor, getSeverityType } from '../../../../../../ml/public'`;

Reply:
sorry about relative path, i copied. it could be different.

Reply:
Thank you for your suggestion. There was some changes in some latest commit in kibana that affected this changes I will update them